test/qpy_compat/test_qpy.py

```diff
@@ -632,8 +715,10 @@ def generate_circuits(version_parts):
     if version_parts >= (0, 24, 1):
         output_circuits["open_controlled_gates.qpy"] = generate_open_controlled_gates()
         output_circuits["controlled_gates.qpy"] = generate_controlled_gates()
-    if version_parts > (0, 24, 2):
+    if version_parts >= (0, 24, 2):
```

Comment:
Oh, good catch yeah now that 0.24.2 has been released this was skipping the layout test from 0.24.2 -> 0.25.0

Reply:
oh, I think I may have done that accidentally - it was part of a merge conflict haha